crates/pallet-domains/src/runtime_registry.rs

```diff
@@ -33,6 +36,12 @@ pub struct RuntimeObject<Number, Hash> {
     pub updated_at: Number,
 }
 
+#[derive(TypeInfo, Debug, Encode, Decode, Clone, PartialEq, Eq)]
+pub struct ScheduledRuntimeUpgrade {
+    pub code: Vec<u8>,
+    pub version: RuntimeVersion,
```

Comment:
Why not extract the version from the code on demand? It's kind of a duplicated state.

Reply:
The intention is not to store the duplicated state in the runtime, a trade-off between the storage and compute, depends on which one you choose. However, it's fine with me if you keep it as is since we don't expect a lot of them.